lib/src/features/home/home_page.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class HomePage extends ConsumerStatefulWidget {
```

Comment:
Please use the Stateless widget instead since it is more straightforward. If possible, use flutter_hooks instead of initState and Stateful Widget.